### ISSUE TITLE
Landing page error

### ISSUE THREAD

**Issue by jackexu** [kcajc/usvisa-ca#1]
It seems like, on the landing page, it can't click the `Continue` button.

Should we change to using XPATH?

```
Sign in page
Landing page
TERM environment variable not set.
Attempt: 1
ERROR!!! Please check user email/password or network
#################################
```

**Comment by LazzzzyJack** [comment]
Currently I don't have an account to validate this. I can look into this later. But if you have a fix feel free to make a PR.

**Comment by MrSusanovo** [comment]
> It seems like, on the landing page, it can't click the `Continue` button.
> 
> Should we change to using XPATH?
> 
> ```
> Sign in page
> Landing page
> TERM environment variable not set.
> Attempt: 1
> ERROR!!! Please check user email/password or network
> #################################
> ```

Add some cool down time before clicking continue.

**Comment by kcajc** [comment]
Should be fixed in #3 or #4.
